test/config-next/wfe2.json

```diff
@@ -28,12 +28,10 @@
       "serverAddresses": ["sa.boulder:19095"],
       "timeout": "15s"
     },
-    "features": {
-      "AllowAccountDeactivation": true,
```

Comment:
Could you land these feature flag deletions as a separate PR ahead of this one?

Reply:
Sure, fixed in 4abec46